src/shogun/base/SGObject.cpp

```diff
@@ -657,27 +573,6 @@ void CSGObject::get_parameter_incremental_hash(uint32_t& hash, uint32_t& carry,
 	}
 }
 
-void CSGObject::build_gradient_parameter_dictionary(CMap<TParameter*, CSGObject*>* dict)
```

Comment:
this we need to keep working somehow as it is used for the GP framework ... so a replacement needs to be written
The signature will probably change as `TParameter` will be removed and we will use the dispatching code that @gf712 wrote